actions/java-gradle-release/action.yaml

```diff
@@ -54,8 +54,8 @@ runs:
     - name: Check out repository
       uses: bakdata/ci-templates/actions/checkout@1.50.2
```

Comment:
please do it for all the the actions referred in here. It makes it easier to follow the actions afterwards